pkg/chartmuseum/server/multitenant/api.go

```diff
@@ -98,6 +98,7 @@ func (server *MultiTenantServer) deleteChartVersion(log cm_logger.LoggingFn, rep
 func (server *MultiTenantServer) uploadChartPackage(log cm_logger.LoggingFn, repo string, content []byte, force bool) (string, *HTTPError) {
 	filename, err := cm_repo.ChartPackageFilenameFromContent(content)
 	if err != nil {
+		// TODO: this is an error if invalid semver, making "EnforceSemver2" useless...
```

Comment:
@scbizu I can take up deprecating that option in a subsequent PR if that is easier so we can get these dependency updates in and unblock other issues. what do you think?

Reply:
Ok,  it is not urgent but since the dependency (helm) will directly introduce the strict semver checking , it will make the behavior inconsistent for users not set the enforce-semver2 flag .  I agree with that this not the key point to block our new release .

Reply:
yep definitely! I can also fix the comments you left on the tests below when I update `uploadChartPackage` to return `400`s if the version is invalid in that PR. Or we could do it here